app/views/WhatYouOwe.scala.html

```diff
@@ -223,96 +239,119 @@ <h3 id="pre-mtd-payments-heading" class="heading-large">@messages("whatYouOwe.pr
+                                        @messages(s"whatYouOwe.over-due.interest.line2",
+                                            charge.documentDetail.interestFromDate.get.toLongDateShort,
+                                            charge.documentDetail.interestEndDate.get.toLongDateShort,
+                                            charge.documentDetail.interestRate.get
+                                        )
+                                        </div>
+                                    </td>
+                                    <td class="govuk-table__cell numeric">@{
+                                        charge.documentDetail.interestOutstandingAmount.get.toCurrency
+                                    }</td>
+                                </tr>
                             }
+                        }
                         </tbody>
                     </table>
                 </div>
                 @if(chargesList.interestOnOverdueCharges) {
-                    <p id="interest-rate-para" class="interest-rate">
-                        @Html(messages("whatYouOwe.over-due.interest-rate",
-                        <a id="interest-rate-link" href="https://www.gov.uk/government/publications/rates-and-allowances-hmrc-interest-rates-for-late-and-early-payments/rates-and-allowances-hmrc-interest-rates" target="_blank">
-                            {messages("whatYouOwe.over-due.interest-rate.link")}</a>
-                        ))
-                    </p>
+                    @*
+                    @p(content = Html(messages("whatYouOwe.sa-note") +
+                            link(link = appConfig.saViewLandPService(utr.get), messageKey = "whatYouOwe.sa-link", id=Some("sa-note-migrated"), target=Some("_blank")).toString()),
+                        id=Some("sa-note-migrated"))
+*@
+
+                    @p(content = Html(messages("whatYouOwe.over-due.interest-rate-1", "") +
+                        link(outerMessage = Some("."), link = "https://www.gov.uk/government/publications/rates-and-allowances-hmrc-interest-rates-for-late-and-early-payments/rates-and-allowances-hmrc-interest-rates",
+                            messageKey = "whatYouOwe.over-due.interest-rate.link", id = Some("interest-rate-link"), target = Some("_blank")).toString() + messages("whatYouOwe.over-due.interest-rate-2")),
+                        id=Some("interest-rate-para"))
+@*
+                    @p {
+                        @messages("whatYouOwe.over-due.interest-rate-1", "")
+                        @link(link = "https://www.gov.uk/government/publications/rates-and-allowances-hmrc-interest-rates-for-late-and-early-payments/rates-and-allowances-hmrc-interest-rates",
+                            messageKey = "whatYouOwe.over-due.interest-rate.link", id = Some("interest-rate-link"), target = Some("_blank"))
+                        @messages("whatYouOwe.over-due.interest-rate-2")
+                    }
+                    *@
```

Comment:
you have removed the original code and put three different commented versions in